managed-ledger/src/test/java/org/apache/bookkeeper/mledger/impl/ManagedLedgerTest.java

```diff
@@ -4232,4 +4232,24 @@ public void testNoCleanupOffloadLedgerWhenMetadataExceptionHappens() throws Exce
         verify(ledgerOffloader, times(0))
             .deleteOffloaded(eq(ledgerInfo.getLedgerId()), any(), anyMap());
     }
+
+    @Test
+    public void testDeleteCurrentLedgerWhenItIsClosed() throws Exception {
+        ManagedLedgerConfig config = spy(new ManagedLedgerConfig());
+        ManagedLedgerImpl ml = spy((ManagedLedgerImpl) factory.open("testDeleteCurrentLedgerWhenItIsClosed", config));
+        assertEquals(ml.ledgers.size(), 1);
+        ml.config.setMaximumRolloverTime(10, TimeUnit.MILLISECONDS);
```

Comment:
It looks the change will not effect the ledger rollover by maximum rollover time, right?